src/main/resources/order-api.yml

```diff
@@ -143,16 +149,23 @@ paths:
             description: Order not found
             schema:
               $ref: '#/definitions/Error'
+          "409":
+            description: Conflict: the order has been already taken by another business
+            schema:
+              $ref: '#/definitions/Error'
           default:
             description: Unexpected error
             schema:
               $ref: '#/definitions/Error'
 definitions:
   Order:
     properties:
-      id:
+      order_id:
         type: string
```

Comment:
order_number may be better if we are going to communicate it to the customer by email, for example.

Reply:
sure, it's a good idea